bundles/org.openhab.binding.miio/src/main/resources/database/chuangmi.plug.v3fw.json

```diff
@@ -1,64 +1,68 @@
 {
-	"deviceMapping": {
-		"id": [
-			"chuangmi.plug.v3",
-			"chuangmi.plug.v3fw",
-			"chuangmi.plug.hmi206",
-			"chuangmi.plug.hmi208"
-		],
-		"channels": [
-			{
-				"property": "power",
-				"friendlyName": "Power",
-				"channel": "power",
-				"type": "Switch",
-				"refresh": true,
-				"ChannelGroup": "",
-				"readmeComment": "If this channel does not respond to on/off replace the model with chuangmi.plug.v3old in the config or upgrade firmware",
-				"actions": [
-					{
-						"command": "set_power",
-						"parameterType": "ONOFF"
-					}
-				]
-			},
-			{
-				"property": "usb_on",
-				"friendlyName": "USB",
-				"channel": "usb",
-				"type": "Switch",
-				"refresh": true,
-				"ChannelGroup": "actions",
-				"actions": [
-					{
-						"command": "set_usb_*",
-						"parameterType": "ONOFFPARA"
-					}
-				]
-			},
-			{
-				"property": "temperature",
-				"friendlyName": "Temperature",
-				"channel": "temperature",
-				"type": "Number",
-				"refresh": true,
-				"ChannelGroup": "",
-				"actions": []
-			},
-			{
-				"property": "wifi_led",
-				"friendlyName": "Wifi LED",
-				"channel": "led",
-				"type": "Switch",
-				"refresh": true,
-				"ChannelGroup": "actions",
-				"actions": [
-					{
-						"command": "set_wifi_led",
-						"parameterType": "ONOFF"
-					}
-				]
-			}
-		]
-	}
+  "deviceMapping": {
+    "id": [
+      "chuangmi.plug.v3",
+      "chuangmi.plug.v3fw",
+      "chuangmi.plug.hmi206",
+      "chuangmi.plug.hmi208"
+    ],
+    "channels": [
+      {
+        "property": "power",
+        "friendlyName": "Power",
+        "channel": "power",
+        "channelType": "power",
+        "type": "Switch",
+        "refresh": true,
+        "ChannelGroup": "",
+        "actions": [
+          {
+            "command": "set_power",
+            "parameterType": "ONOFF"
+          }
+        ],
+        "readmeComment": "If this channel does not respond to on/off replace the model with chuangmi.plug.v3old in the config or upgrade firmware"
+      },
+      {
+        "property": "usb_on",
```

Comment:
Also, why did the formatting change for this file? It is not inconsistent with the other files.

Reply:
I used that file for my testing, and did some intrim reset to undo my changes for the testing.
seems that that also undone the formatting changes...
Applied the formatter once more.